src/shared/utils/reactProdInvariant.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule reactProdInvariant
+ */
+'use strict';
+
+/**
+ * WARNING: DO NOT manually require this module.
+ * This is a replacement for `invariant(...)` used by the error code system
+ * and will _only_ be required by the corresponding babel pass.
+ * It always throw.
+ */
+function reactProdInvariant(code, a, b, c, d, e, f) {
+  var argCount = arguments.length - 1;
+
+  var message = (
+    'React: production error #' + code + '. ' +
```

Comment:
Can we do:

> Minified exception occurred; visit [URL] for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

Or if you think it's important to have the code in their (so people can group their errors better, maybe?) then we could do "Minified React error #123: visit [URL]...".